editor/filesystem_dock.cpp

```diff
@@ -1770,6 +1772,7 @@ FileSystemDock::FileSystemDock(EditorNode *p_editor) {
 	ED_SHORTCUT("filesystem_dock/copy_path", TTR("Copy Path"), KEY_MASK_CMD | KEY_C);
 	ED_SHORTCUT("filesystem_dock/duplicate", TTR("Duplicate..."), KEY_MASK_CMD | KEY_D);
 	ED_SHORTCUT("filesystem_dock/delete", TTR("Delete"), KEY_DELETE);
+	ED_SHORTCUT("filesystem_dock/rename", TTR("Rename"), FOCUS_NONE);
```

Comment:
I'm not sure "FOCUS_NONE" is a valid value for that argument.
If the shortcut is wanted blank you can just remove the last argument.

Reply:
@groud Thanks! I was not sure what I should put there. It's fixed now.